core-bundle/src/Composer/ScriptHandler.php

```diff
@@ -70,7 +70,8 @@ private static function executeCommand(array $cmd, Event $event): void
             $command[] = $verbose;
         }
 
-        $process = new Process($command);
+        // Composer uses an old version of symfony/console which expects a string (see #1956)
+        $process = new Process(implode(' ', $command));
```

Comment:
We could feature detect the version: see https://github.com/symfony/symfony/commit/8895bc1b5bab9af56fbc4458337f83046347bab5

```suggestion
        // Older versions of symfony/process expect a string (see #1956)
        if (method_exists(Process::class, 'setCommandline')) {
            $command = implode(' ', $command);
        }

        $process = new Process($command);
```